File: Java/BottomNavigationViewJava/app/src/main/java/com/example/bottomnavigationviewjava/NotifyFragment.java
package com.example.bottomnavigationviewjava;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.fragment.app.Fragment;

public class NotifyFragment extends Fragment {

    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState){
        ViewGroup rootView = (ViewGroup)inflater.inflate(R.layout.fragment_notify, container, false);

        return rootView;
    }
}

File: Kotlin/BottomNavigationViewKotlin/app/src/main/java/com/example/bottomnavigationviewkotlin/NotifyFragment.kt
package com.example.bottomnavigationviewkotlin

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment

class NotifyFragment : Fragment(){

    companion object{
        fun newInstatnce() : NotifyFragment{
            return NotifyFragment()
        }
    }

    override fun onCreateView(
        inflater: LayoutInflater,
        container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        val view = inflater.inflate(R.layout.fragment_notify, container, false)
        return view
    }
}